src/components/global-header/global-header.stories.mdx

```diff
@@ -0,0 +1,225 @@
+import { useState } from "react";
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import NavigationBar from "../navigation-bar";
+import Box from "../box";
+import VerticalDivider from "../vertical-divider";
+import { Menu, MenuFullscreen, MenuItem, MenuSegmentTitle } from "../menu";
+import GlobalHeader from "./global-header.component";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import LinkTo from "@storybook/addon-links/react";
+import carbonLogo from "../../../logo/carbon-logo.png";
+
+<Meta
+  title="Global Header"
+  parameters={{
+    info: { disable: true },
+    chromatic: { disable: true },
+    docs: {
+      inlineStories: false,
+      iframeHeight: 200,
+    },
+  }}
+/>
+
+# Global Header
+
+Global Header is a wrapper component designed for creating site-wide navigation layouts.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+
+```javascript
+import GlobalHeader from "carbon-react/lib/components/global-header";
+```
+
+## Examples
+
+### Default
+
+Global Header shares similarities with the <LinkTo kind="Navigation Bar" story="default">Navigation Bar component</LinkTo>, but is permanently fixed to the top of the viewport and has a greater z-index than Navigation Bar.
+
+<Canvas>
+  <Story name="default">
+    <GlobalHeader>Example content</GlobalHeader>
+  </Story>
+</Canvas>
+
+### With logo
+
+A custom logo stored as a HTML element or a React component can be rendered by passing it via the `logo` prop. The logo is wrapped in a container that applies style rules for margins and child element height. It is possible to set a custom height for the logo element, but any value that exceeds the height of the Global Header component will be constrained to 40px.
+
+<Canvas>
+  <Story name="with logo">
+    {() => {
+      const Logo = () => <img height={28} src={carbonLogo} alt="Carbon logo" />;
+      return <GlobalHeader logo={<Logo />}>Example content</GlobalHeader>;
+    }}
+  </Story>
+</Canvas>
+
+### Basic menu
+
+In conjunction with the <LinkTo kind="Menu" story="default">Menu component</LinkTo>, the component can be used to create global navigation layouts.
```

Comment:
```suggestion
In conjunction with the <LinkTo kind="Menu" story="default">Menu component</LinkTo>, the `GlobalHeader` component can be used to create global navigation layouts.
```

Just "the component" is a bit ambiguous here, better I think to name the component you're talking about. (It could simply be `GlobalHeader` instead of "the `GlobalHeader` component".)